testonly/hammer/hammer.go

```diff
@@ -514,7 +514,7 @@ func performOp(ctx context.Context, ep MapEntrypointName, prng *rand.Rand, read
 		// TODO(mhutchinson): This mutation method needs to be removed from here.
 		return write(ctx, prng)
 	default:
-		return fmt.Errorf("internal error: unknown entrypoint %s selected for valid request", ep)
+		return fmt.Errorf("internal error: unknown entrypoint %s selected for operation", ep)
```

Comment:
nit to the pre-existing code: Why such a long error text? `"unknown entrypoint %s"` would be totally enough.

Reply:
Agree. Shortened more, but changing this too much will make it inconsistent with the other long errors and then we'll be on a magical whirlwind adventure to get consistency again.